src/renderers/shared/stack/reconciler/ReactUpdateQueue.js

```diff
@@ -253,15 +240,7 @@ var ReactUpdateQueue = {
     enqueueUpdate(internalInstance);
   },
 
-  validateCallback: function(callback, callerName) {
-    invariant(
-      !callback || typeof callback === 'function',
-      '%s(...): Expected the last optional `callback` argument to be a ' +
-      'function. Instead received: %s.',
-      callerName,
-      formatUnexpectedArgument(callback)
-    );
-  },
+  validateCallback: validateCallback,
```

Comment:
Why is this indirection useful? Can we just import it directly in the calling file instead?